elements/lisk-tree/src/constant.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+export const EMPTY_HASH = Buffer.alloc(0);
```

Comment:
Doesn't buffer.alloc(0) just create an empty buffer? Because empty_hash should be the hash of that. `EMPTYHASH=SHA256()`. We could either import `hash` from lisk-cryptography or hard code the value (with a comment).
```suggestion
export const EMPTY_HASH = Buffer.from(e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855, "hex");
```

Reply:
BTW I suggest not to use magic values. Instead, have `hash(Buffer.alloc(0))